server/bin/www

```diff
@@ -87,3 +90,33 @@ function onListening() {
 
   console.log(`Listening on ${bind}`.red.underline.bold);
 }
+
+/**
+ * Authentication and listener for user connection to socket
+ */
```

Comment:
Let's get rid of unnecessary comments

Reply:
Sorry was just trying to match the rest of the file